kubectl-cloudflow/go.mod

```diff
@@ -33,6 +34,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/pkg/errors v0.8.1 // indirect
+	github.com/rayroestenburg/configuration v0.0.0-20200115015912-550403a6bd87
```

Comment:
Do we allow our own repo deps here?

Reply:
I wanted to make sure we can add fixes directly.